R/elem_app_body.R

```diff
@@ -1,4 +1,5 @@
-elem_app_body <- \(data_nodes, data_groups) sidebarLayout(
+elem_app_body <- \(data_nodes, data_groups) fillRow(
   elem_filter_control_panel(data_groups),
-  elem_app_main_panel(data_nodes)
+  elem_app_main_panel(data_nodes),
+  flex = c(2, 10)
```

Comment:
2:10 values shouldn't be hardcoded. And perhaps we could make the control panel a bit wider without going for 3:9 ratio?